Detectors/TRD/pid/src/TRDPIDLinkDef.h

```diff
@@ -20,5 +20,6 @@
 #pragma link C++ class o2::trd::Dummy + ;
 #pragma link C++ class o2::trd::TRDPIDParams + ;
 #pragma link C++ class o2::conf::ConfigurableParamHelper < o2::trd::TRDPIDParams> + ;
+#pragma link C++ class std::vector < TGraph> + ;
```

Comment:
is this really needed?

Reply:
Yes, root only provides native support for st::array<TGraph> not for this.